R/utils-shared.R

```diff
@@ -500,12 +512,6 @@ ApprovalBarYBottom <- function(lim, ylog, reverse) {
 #' @param ratio A scaling ratio to adjust top or bottom of approval bar rectangle.
 #' @return Approval bar, top or bottom y-axis point, in world coordinates.
 ApprovalBarY <- function(lim, ylog = NULL, reverse, ratio) {
-  if (is.null(ylog)) {
-    # presume the semantics of NULL as FALSE, which may or not be correct, but 
-    # prevents the code from terminating here
-    ylog <- FALSE
-  }
-  
```

Comment:
OK, I couldn't get the Boolean expression to work (looks like R doesn't do fast-fail conditional evaluation?), but I added some functionally equivalent syntax.